.github/workflows/push.yml

```diff
@@ -0,0 +1,115 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+# https://github.com/actions/setup-ruby
+# https://github.com/ruby/setup-ruby#versioning
+
+
+name: CI 
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Run test suite
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        ruby: [ '2.6' ] 
+        node: [ '12', '14' ] 
+
+    # use service instead of docker-compose as the `bundle exec rake` command installs Solr & Fedora
+    #env:
+    #  COMPOSE_FILE: docker-compose.lightweight.yml
+
+    services:
+      mariadb:
+        image: mariadb:10.3
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+        options: >-
+          --health-cmd="mysqladmin ping" 
+          --health-interval=10s 
+          --health-timeout=5s
+          --health-retries=3
+        ports:
+          - 3306:3306
+      redis:
+        image: redis
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
```

Comment:
Sometimes it's `--health-interval=10s` and sometimes it is `--health-interval 10s` [like the docs](https://docs.github.com/en/actions/guides/creating-redis-service-containers).  Just a minor nit but could be consistent.